Source/WebCore/dom/ViewTransition.h

```diff
@@ -215,6 +215,8 @@ class ViewTransition : public RefCounted<ViewTransition>, public VisibilityChang
     // ActiveDOMObject.
     void stop() final;
 
+    bool isCrossDocument() { return !m_shouldCallUpdateCallback; }
```

Comment:
Can we rename the member as well? and change the usage in `callUpdateCallback()` ? 

I find this member name a bit confusing, since you could easily confuse it for the case no update callback is provided.